tests/workbenches/notebook_images/upgrade/_kernel_start.py

```
@@ -0,0 +1,58 @@
+    method="POST",
+)
+resp = opener.open(req, timeout=30)
+kernel = json.loads(resp.read())
+kernel_id = kernel["id"]
+resp.close()
+
+time.sleep(2)
+
+conn_file = f"/opt/app-root/src/.local/share/jupyter/runtime/kernel-{kernel_id}.json"
+kc = BlockingKernelClient()
+kc.load_connection_file(connection_file=conn_file)  # noqa: FCN001
```

Ruff (0.15.20)

[warning] Invalid rule code in `# noqa`: FCN001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)